### PR TITLE
Added custom toggles and added some initial ones to the feature-selec…

### PR DESCRIPTION
**Pull request by noahallen**
Added custom toggles and added some initial ones to the feature-selection page.
To test this, run `npm start` and add "/feature-selection" to the end of the URL.

### CODE REVIEW

**Review by CloggedToaster — APPROVED** [review]
Good overall, minor CSS changes

File: src/style/Toggleslider.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/* ToggleSlider.css */
```
[the remark this answers is not in the document]
Comment (noahallen) [reply src/style/Toggleslider.css:1]: Yeah so I agree I think thats a good idea. I think for the toggleslider css its p much only css to get the slider to look the way it does, and wouldn't need to be reused. The only thing may be the color, which I can do the psuedo class for that I suppose

File: src/App.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import Routing from "./routes/index";
 
 function App() {
 	return (
-		<div>
+		<div style={{ padding: "0 10px" }}>
```
[the remark this answers is not in the document]
Comment (noahallen) [reply src/App.js:8]: Moved to App.css

File: src/style/Toggleslider.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (noahallen) [reply src/style/Toggleslider.css:1]: Added for checked color incase we want to match

**Pull request merged**